App/PyImports/Utils/Helpers.py

```diff
@@ -0,0 +1,38 @@
+def dict2xml(d, root_node=None):
+    wrap          =  False if None == root_node or isinstance(d, list) else True
+    root          = 'root' if None == root_node else root_node
+    root_singular = root[:-1] if 's' == root[-1] and None == root_node else root
+    xml           = ''
+    attr          = ''
+    children      = []
+
+    if isinstance(d, dict):
+        for key, value in dict.items(d):
+            if isinstance(value, dict):
+                children.append(dict2xml(value, key))
+            elif isinstance(value, list):
+                children.append(dict2xml(value, key))
+            elif key[0] == '@':
+                attr = attr + ' ' + key[1::] + '="' + str(value) + '"'
+            else:
+                xml = '<' + key + ">" + str(value) + '</' + key + '>'
+                children.append(xml)
+
+    else:
+        #if list
+        for value in d:
+            children.append(dict2xml(value, root_singular))
+
+    end_tag = '>' if 0 < len(children) else '/>'
+
+    if wrap or isinstance(d, dict):
+        xml = '<' + root + attr + end_tag
+
+    if 0 < len(children):
+        for child in children:
+            xml = xml + child
+
```

Comment:
or list comprehension
`xml = [i for i in children]`

Reply:
Done. See 4e60380dfb36f36e53e9c24e17f93143602c5af5